README.md

```diff
@@ -6,58 +6,55 @@
 
 Fylla, the Norse word for `complete`, is an autocompletion script generator for the [Thor](whatisthor.com) framework.
 
-It currently generates zsh completion scripts, but will soon have support for bash scripts as well. 
+Fylla currently generates zsh completion scripts. Support for bash scripts will be added.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+Add the following line to your application's Gemfile:
 
 ```ruby
 gem 'fylla'
 ```
 
-And then execute:
+execute using the following command:
 
     $ bundle
 
-Or install it yourself as:
+Or install fylla manually:
 
     $ gem install fylla
 
 ## Usage
 
-`Fylla` must be loaded before `Thor.start` is called in order for it to be used. 
+`Fylla` must be loaded before `Thor.start` can be used. 
 
 General use case will be to create a new subcommand or option that calls `Fylla.zsh_completion(self)`
 
-`Fylla#zsh_completion` returns a string containing the entire zsh completion script, that you can 
-use to do what you see fit.
+`Fylla#zsh_completion` returns a string containing the entire zsh completion script and bash completion script. Scripts are ready for use after `Fylla#zsh_completion` returns.
```

Comment:
generally in markdown you try to limit lines to 80-120 characters. I see I have failed to do that below, so I'd rather you reformat the below lines than this one.